test/test_jit.py

```diff
@@ -426,10 +426,11 @@ def allSum(vs):
 
         return ge
 
-    def assertAllFused(self, graph):
+    def assertAllFused(self, graph, except_for=()):
```

Comment:
nit: maybe `except_for=frozenset()`? Since sets get passed in for except_for

Reply:
Sure, I could change that, but technically it doesn't have to be a set (only an iterable), since we'll call `set(except_for)` anyway.

Reply:
Oh I see, that's fine then, thanks for the clarification